src/features/titles/utils/index.js

```diff
@@ -0,0 +1,16 @@
+/* global moment */
+
+export const getLetter = index => ( 'abcdefghijklmnopqrstuvwxyz'.toUpperCase() )[ Math.max( 0, Math.min( index, 26 ) ) ];
+
+export const getDateString = date => moment( date ).format( 'MMMM D, YYYY — HH:mm' );
```

Comment:
IIRC there's utils in `wp-date` for this (I think I wrote them) which will respect user's date format settings

Reply:
Was just going off the designs, but may as well use the WP functions 👍

Reply:
Decided to leave this one - those utils don't use the site settings as a default. Can come back to it later.